Add example of no. of entities for scorecards (#1593)
port-labs/port-docs: docs/promote-scorecards/examples.md

    }
  ]
}
```

> ### Validate number of related entities

Author: Agree, fixing

> To assess the deployment frequency of a `Service`, simply checking the `Deployment` relation is not enough—we need to know the exact number of deployments. To achieve this, we will:

Reviewer: ```suggestion
To assess the deployment frequency of a `Service`, simply checking the `Deployment` relation is not enough — we need to know the exact number of deployments. To achieve this, we will:
```

> To assess the deployment frequency of a `Service`, simply checking the `Deployment` relation is not enough — we need to know the exact number of deployments. To achieve this, we will:

Reviewer 2: ```suggestion
To assess the deployment frequency of a `service`, simply checking the `deployment` relation is not enough — we need to know the exact number of deployments. To achieve this, we can:
```

> To effectively track the `Team` deployment frequency and set rules based on it, simply checking the `Deployment` relation to the Team is not enough—we need to know the exact number of deployments. To achieve this, we will:

Author: Agree, fixing and adjusting to "service"

> - Add an [aggregation property](/build-your-software-catalog/customize-integrations/configure-data-model/setup-blueprint/properties/aggregation-property.md) to the `Service` blueprtint that counts the number of related `Deployment` entities.

Reviewer 2: ```suggestion
- Add an [aggregation property](/build-your-software-catalog/customize-integrations/configure-data-model/setup-blueprint/properties/aggregation-property.md) to the `service` blueprint that counts the number of related `deployment` entities.
```

> - Create an [aggregation property](/build-your-software-catalog/customize-integrations/configure-data-model/setup-blueprint/properties/aggregation-property.md) to count the number of related entities (e.g., the number of deployments in the past week).

Reviewer 2: Create an aggregation prop on which blueprint? This example needs a bit of an intro of the scenario

Author: Agree, added

> - Add a scorecard with a rule based on that property.

Reviewer 2: ```suggestion
- Add a scorecard with a rule based on the new aggregation property:
```
